app/views.py

```diff
@@ -11,58 +12,75 @@
 from app.team_profile import get_team_profile
 from app.interested import user_interested
 
-
-@app.route('/user-profile', methods=['GET', 'POST'])
-def user_profile():
-    return update_profile()
+from app.util import format_string
+from app.schemas import ensure_json, ensure_user_logged_in, ensure_feature_is_enabled
 
 
-@app.route('/start-a-team', methods=['POST'])
+@app.route("/user-profile", methods=["GET", "POST"])
+@ensure_json()
+# @ensure_user_logged_in()
```

Comment:
This was commented out for testing locally, but let's uncomment it for the actual PR. One of our tasks moving forward will be to implement tests so that we won't need to rely on stuff like this.